tests/spec/date_series_ops/test_date_comparisons.py

```diff
@@ -11,7 +11,9 @@
         1 | 1990-01-01
         2 | 2000-01-01
         3 | 2010-01-01
-        4 |
+        4 | 2010-01-02
+        5 | 2010-01-03
+        6 |
```

Comment:
I think it would be better to leave this table as it is and define a new one for the new tests. Otherwise you end up cluttering the other tests with examples that don't really add explanatory value for them.

Reply:
I tried this and the documentation generation broke - could you advise as the right way to do this? I've seen other examples where there's a table within the test method, but don't wan't to replicate across all four test methods in this case

Reply:
Ah, that's annoying. Let me have a fiddle around with it and see what works.